code/controllers/subsystem/atoms.dm

```diff
@@ -108,6 +108,19 @@ SUBSYSTEM_DEF(atoms)
 		A.LateInitialize()
 	roundstart_loaders.Cut()
 
+/datum/controller/subsystem/atoms/proc/spawn_static_comms()
```

Comment:
No, that's not an SSatoms thing 😠 
These are core internals, you can't slap stuff like that here